nixos/modules/programs/kdeconnect.nix

```diff
@@ -0,0 +1,24 @@
+{ config, pkgs,lib, ... }:
+with lib;
+{
+  options.programs.kdeconnect = {
+    enable = mkEnableOption "kdeconnect";
+    package = mkOption {
+      default = pkgs.kdeconnect;
```

Comment:
Please add `defaultText = "pkgs.kdeconnect";` after the `default = ...` line. That prevents the nixos manual from depending on pkgs.kdeconnect.